app/scripts/modules/core/src/utils/stickyHeader/stickyHeader.less

```diff
@@ -1,6 +1,12 @@
 @import (reference) "~core/presentation/less/imports/commonImports.less";
```

Comment:
This .less file is all that is left in `/utils/stickyHeader/`.
Anyone have a better place for it?